src/configurations/sources/facebook_lead_ads_native/db-config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "facebook_lead_ads_native",
```

Comment:
@utsabc, let's also remove "native" from all the places to make it less ambiguous.
CC @krishna2020